### ISSUE TITLE
Invalid registrar key "Transcode"

### ISSUE THREAD

**Issue by juliowissing-iis** [CEA-LIST/N2D2#120]
Hello everyone,

I am trying to import an onnx model that has previously been converted from a tf1 .pb file. When I try to test my .ini configuration I get the following error:

Layer: DS_CNN/conv_1/batch_norm/FusedBatchNormV3:0 [Conv]
  Added padding: 1 4  -- 1 5 
Invalid registrar key "Transcode"
Error: bad_function_call

command: n2d2 export_config.ini -seed 1 -w /dev/null -test

.ini: 
```
[database]
Type=DIR_Database
DataPath=database/
ValidExtensions=csv
Learn=0.6
Validation=0.2
Test=0.2

[sp]
SizeX=10
SizeY=49
NbChannels=1

[onnx]
Input=sp
Type=ONNX
File=model_pb.onnx
Ignore=softmax

[import/DS_CNN/logits_fully_connected/BiasAdd.Target]
TopN=5
```

**Comment by cmoineau** [comment]
Hello @juliowissing-iis,

Can you please share your ONNX file to help us investigate on the issue ?

Thanks in advance,
Cyril

**Comment by cmoineau** [comment]
Closing for inactivity.
Feel free to re-open this issue if you still encounter the problem described.
